frontend/src/components/mine/Reports/MineReportTable.js

```diff
@@ -72,6 +74,35 @@ const columns = [
       </div>
     ),
   },
+  {
+    title: "Documents",
+    dataIndex: "documents",
+    render: (text, record) => (
+      <div title="Documents">
+        <ul>
+          {record.report.mine_report_submissions &&
+          record.report.mine_report_submissions.length > 0 &&
+          record.report.mine_report_submissions[record.report.mine_report_submissions.length - 1]
+            .documents.length > 0
+            ? record.report.mine_report_submissions[
```

Comment:
Is this only checking length to either map through the data or show `N/A` ?
that can be done in the initial check above to simplify things. 

If its not doing that, can you add comments to make it more clear

Reply:
This is checking that there are submissions on the report and then if there are submissions does the newest one have documents. If it does display those documents.

Reply:
ohhh, okay